### PR TITLE
swapserverrpc: add module for server protos

### PR DESCRIPTION
**Pull request by carlaKC**
#### Pull Request Checklist
- [x] Update `release_notes.md` if your PR contains major features, breaking changes or bugfixes

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -11,6 +11,7 @@ require (
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/lightninglabs/aperture v0.1.6-beta
 	github.com/lightninglabs/lndclient v0.14.0-5
+	github.com/lightninglabs/loop/swapserverrpc v0.0.0-00010101000000-000000000000
```
Comment (bhandras) [inline go.mod:14]: This could use a future tag (e.g. v1.0.0) we add when the PR is merged right?

**Review by bhandras — APPROVED** [review]
LGTM :+1:

File: go.mod (hunk 1 above)
Comment (guggero) [reply go.mod:14]: Yes, if `go mod` somehow allows it, I'd point this to a tag that doesn't exist yet so we can just create and push it once the PR is merged.

**Review by guggero — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (carlaKC) [reply go.mod]: TIL you can do that!

**Pull request merged**